README.md

```diff
@@ -47,7 +47,23 @@ Follow the steps below to compile and install Soufflé on a UNIX system:
 1.  Install the needed dependencies. G++ 4.8 or greater is recommended to compile Soufflé. openjdk-7 can be used if openjdk-8 is not available on your platform.
 
 ```
-sudo apt-get install autoconf automake bison build-essential doxygen flex g++ git libsqlite3-dev libtool make openjdk-8-jdk pkg-config python zlib1g-dev
+apt-get install -y build-essential;
+apt-get install -y \
+	autoconf \
+	automake \
+	bison \
+	flex \
+	g++ \
+	libsqlite3-dev \
+	libtool \
+	libncurses5-dev \
+	lsb-release \
+	make \
+	openjdk-8-jdk \
+	git \
+	zlib* \
```

Comment:
Why not just zlib1g-dev? #424 suggested the missing libncurses5-dev library was the problem. zlib* can pull in more than is needed, regardless.